java/README.md

```diff
@@ -0,0 +1,12 @@
+Prerequisites
+-------------
+
+* JDK 22
+* Maven 3.9.6 or later
+
+Please build libcuvs (`./build.sh libcuvs` from top level directory) before building the Java API with `./build.sh` from this directory. 
```

Comment:
As per comment above, I think we should mention this, but also mention that calling `build.sh java` from the top level will automatically build libcuvs.